zeebe/scheduler/src/main/java/io/camunda/zeebe/scheduler/future/ActorFuture.java

```diff
@@ -68,6 +70,68 @@ public interface ActorFuture<V> extends Future<V>, BiConsumer<V, Throwable> {
    */
   void onComplete(BiConsumer<V, Throwable> consumer, Executor executor);
 
+  /**
+   * Runs a callback when the future terminates successfully If the caller is not * an actor, the
```

Comment:
:upside_down_face: Something messed with the linebreaks here, there are `*` all over.